markdown/events/2023/hackathon-march-2023/rs-apis-belgrade.md

```diff
@@ -11,9 +11,82 @@ address: Savska 5, Belgrade 11000, Serbia
 location_url: https://startit.rs/beograd/
 location_latlng: [44.80696373983825, 20.45561989400684]
 ---
-
 Local event to be held at the Startit Centar, by Bioinformatics Department of Apis Assay Technologies Ltd.
 
-Primary contact: [<i class="fab fa-slack"></i> Nadezda Bogdanovic](https://nfcore.slack.com/archives/D04RWFB6QP6) or e-mail ([nadezda.bogdanovic@apisassay.com](mailto:nadezda.bogdanovic@apisassay.com)).
+Primary contact: [`<i class="fab fa-slack"></i>` Nadezda Bogdanovic](https://nfcore.slack.com/archives/D04RWFB6QP6) or e-mail ([nadezda.bogdanovic@apisassay.com](mailto:nadezda.bogdanovic@apisassay.com)).
+
+Please use the [main registration form](https://nf-co.re/events/2023/hackathon-march-2023) to sign up, but feel free to email Nadezda if you would like any more information.
+
+# Local Schedule
+
+<div class="table-responsive">
+    <table class="table table-hover table-sm table-bordered">
+        <thead>
+            <tr>
+                <th>Time</th>
+                <th>Mon. 27 Mar., 2023</th>
+                <th>Tue. 28 Mar., 2023</th>
+                <th>Wed. 29 Mar., 2023</th>
+            </tr>
+            </thead>
+            <tbody>
+            <tr>
+                <td data-timestamp="1679907600" data-timeformat="HH:mm z">09:00 CET</td>
+                <td background-color:navy; rowspan="1">Local Welcome and Setup</td>
```

Comment:
```suggestion
                <td style="color: pearl; background-color:navy;" rowspan="1">Local Welcome and Setup</td>
```
Otherwise the styling is not applied

Reply:
Changed